media-libs/smpeg/smpeg-0.4.4-r11.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools flag-o-matic multilib-minimal
+
+DESCRIPTION="SDL MPEG Player Library"
+HOMEPAGE="https://icculus.org/smpeg/"
+SRC_URI="https://mirrors.dotsrc.org/lokigames/open-source/smpeg/${P}.tar.gz
+	mirror://gentoo/${P}-gtkm4.patch.bz2"
+
+LICENSE="LGPL-2+"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux ~x86-linux ~x86-solaris"
+IUSE="X debug cpu_flags_x86_mmx opengl"
```

Comment:
I was wondering if this needed xorg-proto (most cases libX11 is used, you need it in DEPEND)... but interestingly X does nothing.

It checks for headers during configure but then doesn't include them, doesn't link with libraries, and the produced binaries are also perfectly identical on or off.

So I'll drop IUSE=X on merge and use a hard --without-x so it doesn't check for nothing. I assume it's some vestige from a previous version.